src/transformers/models/auto/configuration_auto.py

```diff
@@ -625,6 +627,7 @@
         ("roberta-prelayernorm", "RoBERTa-PreLayerNorm"),
         ("roc_bert", "RoCBert"),
         ("roformer", "RoFormer"),
+        ("rt_detr", "RTDetr"),
```

Comment:
```suggestion
        ("rt_detr", "RT-DETR"),
```
If you run `make fix-copies`, it will update the index table etc